src/input_xml.F90

```diff
@@ -2489,6 +2494,102 @@ subroutine read_tallies_xml()
             ! Set to analog estimator
             t % estimator = ESTIMATOR_ANALOG
 
+          case ('mu')
+            ! Set type of filter
+            t % filters(j) % type = FILTER_MU
+
+            ! Set number of bins
+            t % filters(j) % n_bins = n_words - 1
+
+            ! Allocate and store bins
+            allocate(t % filters(j) % real_bins(n_words))
+            call get_node_array(node_filt, "bins", t % filters(j) % real_bins)
+
+            ! Allow a user to input a lone number which will mean that
+            ! you subivide [-1,1] evenly with the input being the number of bins
+            if (n_words == 1) then
+              Nangle = abs(int(t % filters(j) % real_bins(1)))
```

Comment:
Ah, good catch. Used to have a negative signify when the user was inputting a number of bins and that was a remnant from those dark dreary days